tools/redfish-gen/redfish/v1/Systems/System.yaml

```diff
@@ -0,0 +1,194 @@
+                  - Name: HealthRollup
+                    SourceField: StatusProvider::fieldStatusRollup
+                  - Name: State
+                    SourceField: Server::fieldState
+            Enums:
+              - &HealthEnum
+                Name: Health
+                Source: StatusProvider::Status
+                Mapping:
+                  OK: ok
+                  Warning: warning
+                  Critical: critical
+              - Name: State
+                Source: Server::State
+                Mapping:
+                  Enabled: enabled
+          - Name: Boot
+            Entity:
+              - Source: Server
+                Fields:
+                  - Name: BootSourceOverrideEnabled
+                    SourceField: Server::fieldBootOverrideType
+                  - Name: BootSourceOverrideMode
+                    SourceField: HostBootProvider::fieldType
+                  - Name: BootSourceOverrideTarget
+                    SourceField: HostBootProvider::fieldMode
+                  - Name: AutomaticRetryConfig
+                    SourceField: BootSettingsProvider::fieldRetryConfig
+                  - Name: AutomaticRetryAttempts
+                    SourceField: RebootRemainingProvider::fieldAttemptsLeft
+            Enums:
+              - Name: AutomaticRetryConfig
+                Source: BootSettingsProvider::RetryConfig
+                Mapping:
+                  Disabled: disabled
+                  RetryAttempts: retryAttempts
+              - Name: BootSourceOverrideEnabled
+                Source: Server::BootOverrideType
+                Mapping:
+                  Disabled: none
+                  Once: once
+                  Continuous: continuous
+              - Name: BootSourceOverrideMode
+                Source: HostBootProvider::Type
+                Mapping:
+                  Legacy: legacy
+                  UEFI: uefi
+              - Name: BootSource
+                For: BootSourceOverrideTarget
+                Source: HostBootProvider::Mode
+                Mapping:
+                  None: none
+                  Pxe: pxe
+                  Cd: cd
+                  Usb: usb
+                  Hdd: hdd
+                  BiosSetup: biosSetup
+                  Diags: diags
+          # - Name: ProcessorSummary
+          #   Entity:
+          #     - Source: ProcessorSummary
+          #       Fields:
+          #         - Name: Count
+          #           SourceField: ProcessorSummary::fieldCount
+          #   Fragments:
+          #     - Name: Status
+          #       Entity:
+          #         - Source: ProcessorSummary
+          #           Fields: *Health
+          #       Enums:
+          #         - *HealthEnum
+          #         - Name: State
+          #           Source: Processor::State
+          #           Mapping:
+          #             Enabled: enabled
+          #             Absent: absent
```

Comment:
Do we really need to keep this in git?

Reply:
It is my mistake, I was  comment this for the debugging purpose to avoid initialize dbus-cache that is not required to test 'in place now'. It must be dis commented.